### PR TITLE
Update IoTHub SDK

### PR DESCRIPTION
**Pull request by anusapan**
Add test for manual-failover.
fixed broken test
[Swagger Link](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json)

### CODE REVIEW

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/generate.ps1 [hunk 1]
```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "iothub/resource-manager" -AutoRestVersion "latest"
+Start-AutoRestCodeGeneration -ResourceProvider "iothub/Microsoft.Azure.Management.IotHub" -AutoRestVersion "latest"
```
Comment (erich-wang) [inline sdk/iothub/Microsoft.Azure.Management.IotHub/src/generate.ps1]: Any particular reason to change it?

**Review by erich-wang — CHANGES_REQUESTED** [review]
1. Please provide swagger spec PR link related to this PR
1. Please follow **step 8** in [instruction](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-directory-structure-and-sdk-generation). It will really help us a lot to review your PR if it contains sdk\_metadata\<rpName>.txt which is generated by generate.ps1

1. Please follow the [instruction](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#creating-pr)
> If **ExcludeFromTest** and **ExcludeFromBuild** are present in your test csproj, make sure their value are false, otherwise you tests won't be run from CI

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/generate.ps1 (hunk 1 above)
Comment (anusapan) [reply sdk/iothub/Microsoft.Azure.Management.IotHub/src/generate.ps1]: change reverted

**Comment by anusapan** [comment]
@erich-wang Is this PR still in the review process?

**Comment by erich-wang** [comment]
@anusapan , next time please provide swagger spec PR link in this PR description which looks like https://github.com/Azure/azure-rest-api-specs/pull/xxxx, otherwise I don't know what is changed in the swagger spec and whether it is approved.
> 1. Please provide swagger spec PR link related to this PR

**Review by erich-wang — APPROVED** [review]

**Comment by anusapan** [comment]
@erich-wang Could you tell me, How can I signed/release the NuGet package of iothub?

**Comment by erich-wang** [comment]
@anusapan , the process is available at https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#signing-and-publishing-nuget-packges

**Pull request merged**